src/glib/base/gix.h

```diff
@@ -569,7 +569,10 @@ void TGixItemSet<TKey, TItem, TGixMerger>::AddItem(const TItem& NewItem, const b
         }
         // TODO: why is next RecalcTotalCnt needed? Def() already calls it if anything is changed. It might be needed only if IsFull() was true.
         RecalcTotalCnt(); // work buffer might have been merged
-        Gix->AddToNewCacheSizeInc(GetMemUsed() - OldSize);
+        const uint64 NewSize = GetMemUsed();
```

Comment:
Would it be possible to move this code inside AddToNewCacheSizeInc to remove duplications of checking for size, etc.

In this case AddToNewCacheSizeInc would take two parameters, old sizde and new size, and it would update the increment according to the difference, same as is done now here?